packages/components/releaseNotes/labkey/components.md

```diff
@@ -1,6 +1,10 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version TBD
+*Released*: TBD
+* Support error handling for inaccurately inferred field types within List and Dataset designers.
```

Comment:
can you add in the issue number to this note?

Reply:
Done!